lib/dynamodb_framework/dynamodb_table_manager.rb

```diff
@@ -257,6 +257,7 @@ def create(table_name, attributes, partition_key, range_key = nil, read_capacity
           :table_name => table_name,
           :attribute_definitions => attribute_definitions,
           :key_schema => key_schema,
+          :billing_mode => 'PAY_PER_REQUEST',
```

Comment:
This needs to be an option passed into the method, it shouldn't default to `PAY_PER_REQUEST`. The default in the method signature should be `PROVISIONED`